tests/e2e/tests/web-components/checkbox-group.js

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const test = require('ava');
+const {withPage, addPageScript} = require('../../puppeteer');
+const {html} = require('common-tags');
+
+test(
+  'checkbox group: can restrict the number of items shown',
```

Comment:
I'm quite sure thiese should be unit tests, not e2e tests? I think the gist of our e2e tests is that it requires a build step (running 11ty). Please move it to https://github.com/GoogleChrome/developer.chrome.com/tree/main/tests/site/_js (or tell me if I'm wrong here)

Reply:
Fair enough - I think that makes sense. I was kind of on the fence since were testing in browser/from a user perspective but also understand we are testing a single unit rather than the end product as a whole.

Unless i hear otherwise, I'll move the puppeteer/rollup stuff to the root test directory (so that it can be used by any test type if needs be) and the tests themselves to the /site/_js dir as mentioned.